EpiResponsivePicture/Extensions/ApplicationBuilderExtensions.cs

```diff
@@ -13,12 +18,16 @@ public static class ApplicationBuilderExtensions
     /// </summary>
     /// <param name="app"></param>
     /// <param name="providerRegistration"></param>
-    public static IApplicationBuilder UseForteEpiResponsivePicture(this IApplicationBuilder app, Func<IApplicationBuilder, IApplicationBuilder> providerRegistration = null)
+    public static IApplicationBuilder UseForteEpiResponsivePicture(this IApplicationBuilder app, 
+        Func<IApplicationBuilder, IApplicationBuilder> providerRegistration = null)
     {
-
         providerRegistration ??= SixLabors.ImageSharp.Web.DependencyInjection.ApplicationBuilderExtensions.UseImageSharp;
-        
         providerRegistration(app);
+
+        var imageProvider = app.ApplicationServices.GetServices<IImageProvider>()
```

Comment:
how about `var imageProvider = app.ApplicationServices.GetService<BlobImageProvider>();` ?

Reply:
That won't work, as `BlobImageProvider` is not registered directly as class instance.

Reply:
ok, so one last thing - maybe it's better to do FirstOrDefault() and throw exception saying that there's no service registered for BlobImageProvider? Default exception (sequence contains no elements) might be hard to debug

Reply:
Fixed as suggested.